keycloak/sms-provider/src/main/java/org/sunbird/keycloak/KeycloakSmsAuthenticatorConstants.java

```diff
@@ -21,4 +21,7 @@
     // User credentials (used to persist the sent sms code + expiration time cluster wide)
     public static final String USR_CRED_MDL_SMS_CODE = "sms-auth.code";
     public static final String USR_CRED_MDL_SMS_EXP_TIME = "sms-auth.exp-time";
+
+    public static final String AMAZON_SNS_PROVIDER_CONFIGURATIONS_PATH = "sms-provider/awsSnsCreds.json";
```

Comment:
Are these constants needed ? Can't the files be named in a predictable manner and kept in a predictable location so that you can construct the path dynamically.

eg: configuration/<provider name>_config.json